### PR TITLE
convencaye commands

### PR DESCRIPTION
**Pull request by YoniKiriaty**
dont merge until all elbow , pivot, command aprovved, wrist

### CODE REVIEW

**Comment by aaa-png** [comment]
Yoni - what is this name

**Review by aaa-png — APPROVED** [review]
Yoni

**Review by aaa-png — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import edu.greenblitz.robotName.subsystems.shooter.Pivot.PivotConstants;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class MoveShooterToAngle extends ConditionalCommand {
+
+	public MoveShooterToAngle(Rotation2d targetAngle) {
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java:17]: consider splitting this

**Review by bossyshrimp68 — DISMISSED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java (hunk 1 above)
Comment (YoniKiriaty) [reply src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java:17]: there is no reason for there to be a command that is split from this.

Comment (YHerm) [reply src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java:17]: We would not use the command inside in any other situation, 
so I don't ✖️ see 👁️  a reason to get it a special 🎊 class 🧑‍🏫 just for it

**Review by itamaroryan — APPROVED** [review]

**Review by itamaroryan — APPROVED** [review]
🧽🧽🧽

**Review by aaa-png — APPROVED** [review]

**Pull request merged**